aebs:
  ros__parameters:
    robot_width: 0.44
<<<<<<< HEAD
    safety_margin: 0.18
    ttc_threshold: 0.5
=======
    ttc_threshold: 1.1
    aebs_mode: "tunnel"  
    safety_factor: 2.0
>>>>>>> origin/main
